CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing to Panda3D
+Thank you for looking into ways to contribute to Panda3D.
+Most information on how to contribute and get involved already exists in other places, so this page serves as another way to find those resources.
+
+## Useful Links
+* [panda3d.org](https://www.panda3d.org)
+* [Manual](https://docs.panda3d.org/1.10/python/index)
+
+### Getting in Touch With the Community
+* [Discord](https://discord.gg/UyepRMm)
+* [IRC](https://www.panda3d.org/manual/?title=The_IRC_Channel)
+* [Forum](https://discourse.panda3d.org/)
+
+
+## Building the Project
+Information on how to build Panda3D can be found in the [README](https://www.panda3d.org/get-involved/).
+
+## Testing the Project
+Information on how to test Panda3D can be found in the [README](https://www.panda3d.org/get-involved/).
```

Comment:
I believe you wanted this to point to the readme and not the get involved page.

Reply:
Fixed.